### ISSUE TITLE
Dataviz users: improvements

### ISSUE THREAD

**Issue by DivadNojnarg** [devOpifex/shinymetrics#11]
For now, the dashboard gives a table with 3 columns:

User ID // Sessions // Avg. Session duration

Maybe merge extra info like location, browser, ... a network like viz with nodes and edges ...?

**Comment by JohnCoene** [comment]
That's available on the focus. I'm not sure it would bring a lot of value, particularly "browser"

**Comment by DivadNojnarg** [comment]
I'd like to get your input on this. How do you treat potentially non meaningful sessions (Say someone staying less than 10 seconds without encoutering any errors)? 

<img width="987" alt="Screenshot 2023-07-18 at 09 30 36" src="https://github.com/devOpifex/shinymetrics/assets/18291543/6b575d73-e607-42ad-8227-fc278d8297f8">

I would say that Sessions and Users could be merged together because within the user tab, I'd like to see the average session duration across all users (available in the Session tab). This would lighten the navigation.

**Comment by DivadNojnarg** [comment]
>You can search by user. I will add a table with top sessions.
Would be good so I don't have to go back to the user tab to look for their ID.

I do see some redundancy in the features. For instance, going to `Focus -> entering user ID -> select Session`  gives me the same as going in `Sessions -> Focus -> entering user ID`. Same if instead I do `Focus -> entering user ID -> select User`  VS browse to `Users -> Focus -> user ID`.

**Comment by JohnCoene** [comment]
Yes, this is similar but totally identical. I don't really see a problem with that personally, I think they all have their place. Moreover, the Focus tab may reduce data-processing costs for me and speed things up for the user. But it still makes sense to be able to focus on a session under the "Sessions" tab and focus on a user on the "Users" tab.

Just so I understand: Are you interested in browsing a specific session? If so why? (so I can maybe find a better way to sort through sessions).

Users (if [set](https://pkg.shinymetrics.com/articles/users.html)) can bring valuable information and are easy for the user to parse. This is not true of sessions: user "David" is understandable, session "026812c2ff2e48a82a1c217728adf49b" doesn't mean much.
Ideally there would be a better way to identify sessions (doubt it) but maybe I can do something so it's easier to browse/search. 

For this, I'd like to know more about when a user searches for a specific session and why. Or the reason why one might want to explore individual session.

Not sure I express myself clearly

**Comment by DivadNojnarg** [comment]
> Just so I understand: Are you interested in browsing a specific session? If so why? (so I can maybe find a better way to sort through sessions).

For a long session, I'd like to see why they spent some much time. For short session as well (why did they give up?).

Why do I see 0s sessions? If I inspect them, I don't see anything. Maybe they should be totally excluded, unless there is a good reason like an error but that would likely take more than 0s to happen.

c77f8e5ca596be61044f2145b247fcff - smu_l2xycr0votgwulb5lz0sx0ndxz8ebl  0 sec.
8f10c8881efbf8e5908a8b5a76361bb0 - smu_7nb6vh5xruuunzmo7tbcqnuchznjhd 0 sec.

**Comment by JohnCoene** [comment]
Yes, I noticed that too when I added the table. I'm not sure what it is just yet though I guess for a public it could happen? E.g.: crawlers causing that?

**Comment by JohnCoene** [comment]
Maybe it's just worth filtering them out then showing another table with "worst sessions"

But session duration is one thing, I'm thinking maybe number of events recorded would be a better metric? 

You keep a tab open and barely do anything to have a 30 min long session?

Feedback/opinion welcome! :)

**Comment by DivadNojnarg** [comment]
> But session duration is one thing, I'm thinking maybe number of events recorded would be a better metric?

Yes if the session is 30 min but with 2 events, I agree this is likely not relevant and should be filtered.
